experimental/addons/istio/resources/istio-vs.cue

```diff
@@ -0,0 +1,152 @@
+"istio-vs": {
+	annotations: {}
+	attributes: {
+		appliesToWorkloads: ["*"]
+		podDisruptive: false
+	}
+	description: "Handle Traffic Via Istio VirtualService."
+	labels: {}
+	type: "trait"
+}
+
+template: {
```

Comment:
Should we add destination rule in this trait？

Reply:
IMO this cue definition is a pure istio virtualservice, I can add another cue def for destination rules of course .

Reply:
Ok, feel free to add dr trait in next pr.